cluster-up/cluster/kind-k8s-sriov-1.17.0/sriov-components/manifests/kustomization.yaml

```diff
@@ -8,10 +8,10 @@ resources:
 - sriovdp-config.yaml
 images:
   - name: nfvpe/sriov-device-plugin
-    newName: quay.io/kubevirtci/sriov-device-plugin
+    newName: nfvpe/sriov-device-plugin
```

Comment:
Leftovers from a rebase that went not correct, please check it

same below

Reply:
Removed.